shared/wallets/confirm-form/container.tsx

```diff
@@ -48,13 +48,15 @@ export default namedConnect(
   (stateProps, dispatchProps, _: OwnProps) => {
     const {_built, _sentPaymentError} = stateProps
     const exchangeRateChanged = _sentPaymentError && _sentPaymentError.toLowerCase().includes('exchange rate')
+    const recipientRequiresAMemo =
+      _sentPaymentError && _sentPaymentError.toLowerCase().includes('recipient requires a memo')
```

Comment:
if there are special constants maybe we should make it more explicit as a const above?